style/calendar.less

```diff
@@ -158,29 +191,83 @@
   }
 }
 
-.reading{
-  background: fade(@brand-success, 20%);
-  border-color: @brand-success;
+.plan(@type) {
+  @plan-color: ~"@{type}-color";
+
+  background: fade(@@plan-color, 20%);
+  border-color: @@plan-color;
 
-  &:hover, &.active{
-    background: fade(@brand-success, 75%);
+  &:hover, &.active, &.open {
+    background: fade(@@plan-color, 75%);
   }
-}
 
-.exercise{
-  background: fade(@brand-primary, 20%);
-  border-color: @primary;
+  &-modal {
+    .modal-content {
+      .modal-header {
+        color: @@plan-color;
+        background: fade(@@plan-color, 20%);
+      }
+    }
 
-  &:hover, &.active{
-    background: fade(@primary, 75%);
+    .btn:not(.btn-link):not(.btn-flat) {
+      border: 1px solid @@plan-color;
+      color: @@plan-color;
+    }
   }
 }
 
-.essay{
-  background: fade(@brand-warning, 20%);
-  border-color: @brand-warning;
+.plan-modal {
+  .modal-content {
+    .modal-header {
+      padding-bottom: 10px;
+    }
 
-  &:hover, &.active{
-    background: fade(@brand-warning, 75%);
+    .modal-body {
+      > :first-child {
+        margin-top: 0;
+      }
+    }
+
+    .btn:not(.btn-link):not(.btn-flat) {
+      margin-top: 0;
+      margin-bottom: 0;
+    }
+  }
+
+  .panel {
+    margin: 0;
+    box-shadow: none;
+  }
+
+  .panel-body {
+    padding: 0;
+  }
+
+  .reading-stats {
+    section {
+      > div {
+        max-width: @progress-bar-width;
+      }
+    }
   }
 }
+
+.reading {
```

Comment:
Can these be scoped inside a common container? that way the do not accidentally leak into styling elsewhere? (like a student task step)